cfg/rke-cis-1.24/master.yaml

```diff
@@ -272,7 +272,26 @@ groups:
 
       - id: 1.1.19
         text: "Ensure that the Kubernetes PKI directory and file ownership is set to root:root (Automated)"
-        audit: "check_files_owner_in_dir.sh /node/etc/kubernetes/ssl"
```

Comment:
we 3 code places where I see this script usage. 
can we try making it available as script and copy them in docker image.